chart/open-feature-operator/values.yaml

```diff
@@ -92,6 +110,8 @@ controllerManager:
 managerConfig:
   ## @param managerConfig.flagsValidatonEnabled Enables the validating webhook for FeatureFlag CR.
   flagsValidatonEnabled: "true"
+  ## @param managerConfig.flagdResourceEnabled Enables the controller for the Flagd CR and adds the required permissions to create/update/delete Service and Ingress resources.
```

Comment:
I guess this is the real source of truth for my comment [here](https://github.com/open-feature/open-feature-operator/pull/633/files?diff=split&w=1#r1598677911).